homework-projects/CharlesDube/PhotoColage Part 2.adoc

```diff
@@ -0,0 +1,20 @@
+[[PhotoCollage]]
+== Photo Collage
+
+[[Description]]
+[.lead]
+Photo collage is a graphical tool/application used to gather a user's image gallery data and make collage posters according to the user's design or the results of the inbuilt algorithm.
+
+[[Benefits]]
+There are many difficulties in creating poster designs, collage layouts, and managing proportions in imagery such that users are attracted to your works from clever use of color palettes, sizing, and layouts. PhotoCollage's algorithms solve these problems for the user, streamlining the process, allowing for easy use of the application yet still providing results for the commercial market.
+
+[[HowToUse]]
+=== How to use
+
+1. Download the installation files from link:https://github.com/adrienverge/PhotoCollage[PhotoCollage Repository] 2.1. On Fedora run 'sudo dnf install PhotoCollage' in the terminal 2.2. On Debian 9+ run 'sudo apt-get install photocollage' 2.3. On pip Python manager run 'sudo pip3 install photocollage'
```

Comment:
it's an incorrect asciidoc syntax for numbered list. See more details: https://docs.asciidoctor.org/asciidoc/latest/lists/ordered/